doc/source/notebooks/manual/advanced/multiclass_classification.ipynb

```diff
@@ -4,10 +4,424 @@
+   "metadata": {},
+   "source": [
+    "The multiclass classification problem is a regression problem from an input $x \\in {\\cal X}$ to discrete labels $y\\in {\\cal Y}$ where ${\\cal Y}$ is a discrete set of size $C$ bigger than 2 (for $C=2$ it is the more usual binary classification)\n",
+    "\n",
+    "Labels are encoded in a one-hot fashion, that is if $C=4$ and $y=2$, we note $\\bar{y} = [0,1,0,0]$\n",
+    "\n",
+    "The generative model for this problem we consider is built of \n",
+    " * $C$ latent functions $\\mathbf{f} = [f_1,...,f_C]$ with independent Gaussian Process prior\n",
+    " * a determinisctic function building a discrete distribution $\\pi(\\mathbf{f}) = [\\pi_1(f_1),...,\\pi_C(f_C)]$ from the latents such that $\\sum_c \\pi_c(f_c) = 1$. \n",
+    " * a Discrete Likelihood $p(y|\\mathbf{f}) = Discrete(y;\\pi(\\mathbf{f})) = \\prod_c \\pi_c(f_c)^{\\bar{y}_c}$\n",
+    "\n",
+    "A typical example of $\\pi$ is the softmax function \n",
+    "\n",
+    "$$ \\pi_c (f_c) \\propto \\exp( f_c)$$\n",
+    "\n",
+    "Another convenient one is the Robust Max\n",
+    "$$\n",
+    "\\pi_c(\\mathbf{f}) = \\begin{cases} 1 - \\epsilon, & \\mbox{if } c = \\arg \\max_c f_c \\\\\n",
+    " \\epsilon /(C-1), & \\mbox{ otherwise} \\end{cases}\n",
+    "$$\n",
+    "\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-06-13T09:42:54.989725Z",
+     "start_time": "2018-06-13T09:42:53.821801Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import gpflow\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from gpflow.test_util import notebook_niter, is_continuous_integration\n",
+    "plt.style.use('ggplot')\n",
+    "%matplotlib inline\n",
+    "from multiclass_classification import plot_posterior_predictions\n",
+    "np.random.seed(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sampling from the GP multiclass generative model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Declaring model parameters and input"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-06-13T09:42:55.175406Z",
+     "start_time": "2018-06-13T09:42:54.990770Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "C, N = 3, 100  # number of functions and number of data points\n",
+    "lsc = 0.01  # RBF kernel lengthscale \n",
+    "jitter = 1e-6  # jitter to stabilize sampling\n",
+    "X = np.random.rand(N,1)  # input "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Sampling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "K = np.exp(-0.5*np.square(X - X.T)/lsc) + np.eye(N)*jitter  # RBF kernel matrix\n",
```

Comment:
I don't think this reads as easy as intended.
I'd propose something in the line of
```
kern = gpflow.kernels.RBF(1, lengthscales=lsc)
K = kern.compute_K_symm(X)
f = np.random.multivariate_normal(mean=np.zeros(N), cov=K, size=(C)).T
```

Reply:
`K = kern.compute_K_symm(X) + np.eye(N)*jitter`